pkg/services/accesscontrol/actest/common.go

```diff
@@ -113,8 +114,15 @@ func AddUserPermissionToDB(t testing.TB, db db.DB, user *user.SignedInUser) {
 		var permissions []accesscontrol.Permission
 		for action, scopes := range user.Permissions[user.OrgID] {
 			for _, scope := range scopes {
+				kind, uid := "", ""
+				parts := strings.Split(scope, ":")
+				if len(parts) == 3 && parts[1] == "uid" {
+					kind, uid = parts[0], parts[2]
+				}
 				p := accesscontrol.Permission{
-					RoleID: role.ID, Action: action, Scope: scope, Created: time.Now(), Updated: time.Now(),
+					RoleID: role.ID,
+					Action: action, Scope: scope, Kind: kind, Identifier: uid,
+					Created: time.Now(), Updated: time.Now(),
 				}
```

Comment:
[suggestion] Consider creating a helper function to map a `map[string][]string` to `[]accesscontrol.Permission` for consistency.

Reply:
In general, for such tests we should have a helper function to construct an `accesscontrol.Permission`. However as search is being rewritten - all these tests will probably get rewritten anyway, so I kept it verbose and explicit to keep the changes minimal.